metasrv/src/store/meta_raft_store.rs

```diff
@@ -101,6 +101,8 @@ pub struct MetaRaftStore {
 
     /// The current snapshot.
     pub current_snapshot: RwLock<Option<Snapshot>>,
+
+    pub endpoint: Endpoint,
```

Comment:
The endpoint is already stored in the `state_machine`: `MetaRaftStore::get_node_endpoint()`. Is there another purpose for which we need store endpoint?